### PR TITLE
Add notes from first discussion

### PR DESCRIPTION
**Pull request by jarrodmillman**
This needs editing and review before merging.  I also need to link to the video once it is made public.

### CODE REVIEW

File: source/discussion/2021-12-15.rst [hunk 1]
```diff
@@ -0,0 +1,268 @@
+Dispatching discussion - 12/15/21
+=================================
+
+-  Stefan (UC Berkeley)
```
Comment (seberg) [inline source/discussion/2021-12-15.rst]:
```suggestion
-  Stefan van der Walt (UC Berkeley)
```

File: source/discussion/2021-12-15.rst [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+      maintainers
+   -  Minimize duplication of pure-Python code by backend implementers
+   -  Combine ``array_api`` with a function-level dispatcher to support
+      most use-cases
+
+-  Having ``scikit-learn`` be the API is viewed as a positive, +1 for
+   dispatching
+
+-  
+
```
Comment (seberg) [inline source/discussion/2021-12-15.rst]:
```suggestion
```

File: source/discussion/2021-12-15.rst [hunk 3]
```diff
@@ -0,0 +1,268 @@
...
+-  Have some way for backend-implementers to indicate back to users how
+   much of a given API they support. This usually isn’t immediately
+   obvious to users. Whose job is it to inform users how much of any
+   given API is supported by a backend. Can be addressed via tooling
+
+   -  ``numba`` has a similar problem (currently manually updated), but
+      some tooling for this
+   -  Similar problem in ``dask`` via ``__array_function__``
+   -  Libraries may be able to reject backend registration if it’s known
```
Comment (seberg) [inline source/discussion/2021-12-15.rst]:
```suggestion
   -  Libraries may want to be able to reject backend registration if it’s known
```

File: source/discussion/2021-12-15.rst [hunk 4]
```diff
@@ -0,0 +1,268 @@
...
+
+-  Explicitly layout the target audience, potential (hoped-for?) impact?
+
+-  Organize another call - open to the public
+
+Afternotes
+----------
+
+-  single-dispatch (stlib) vs multi-dispatch
+-
```
Comment (seberg) [inline source/discussion/2021-12-15.rst]:
```suggestion
```

**Review by seberg — COMMENTED** [review]
There could probably be clarifications to some bullets, but I had a brief read-through, and I think this is fine to publish (and can't think of important things discussed and missing here).

**Comment by jarrodmillman** [comment]
@rgommers Should we merge this now?  And then you can update it in a subsequent PR?

@juanis2112 uploaded the cleaned up video and I updated (in f01f139) the link to the video in the meeting notes.

**Comment by jarrodmillman** [comment]
I don't like that I turn the camera down at the end of the video.  Any thoughts about how to remove that part?  Would fading to an final slide with a link to the website to cover up my mistake be reasonable?

https://api-dispatch.scientific-python.org/discussion/2021-12-15.html
or
https://api-dispatch.scientific-python.org/

The first link won't exist until this PR is merged.

**Comment by rgommers** [comment]
I fixed the affiliations, I'm fairly sure they're correct now.

> I don't like that I turn the camera down at the end of the video. Any thoughts about how to remove that part?

Maybe that's already fixed? I don't see it in the video. Overall the video looks really good to me!


> @rgommers Should we merge this now? And then you can update it in a subsequent PR?

Makes sense to me. You did say right at the end of the video that you were going to send around a link to the recording so people could check. I just watched most of the recording, and I don't think anyone said anything they'd have second thoughts about, but I just thought I'd point that out.

**Comment by jarrodmillman** [comment]
I emailed the participants and told them we would wait until Wednesday (1/12) before merging this.

File: source/discussion/2021-12-15.rst [hunk 5]
```diff
@@ -0,0 +1,270 @@
...
+
+- Stéfan van der Walt (NumPy, scikit-image, UC Berkeley)
+- Ralf Gommers (NumPy, SciPy, Quansight)
+- Sebastian Berg (NumPy, UC Berkeley)
+- Jarrod Millman (NetworkX, UC Berkeley)
+- Ross Barnowski (NumPy, UC Berkeley)
+- Ivan Yashchuk (PyTorch, Quansight)
+- John Kirkham (Dask, NVIDIA)
+- Leo Fang (CuPy, NVIDIA)
+- Kenichi Maehashi (CuPy, Preferred Networks)
```
Comment (leofang) [inline source/discussion/2021-12-15.rst]: I don't think Kenichi attended.

Comment (rgommers) [reply source/discussion/2021-12-15.rst]: Thanks, good catch. He was invited but didn't attend (time zones ...)

**Pull request merged**